source/infrastructure/pipeline/e2e-tests/utils/schedule-test-utils.ts

```diff
@@ -17,14 +17,19 @@ export interface Schedule {
   description: string;
   periods: Period[];
 }
-export async function createSchedule(client: dynamodb.DynamoDBClient, schedule: Schedule) {
-  await client.send(
-    new dynamodb.BatchWriteItemCommand({
-      RequestItems: {
-        [configTableName]: scheduleToBatchedPutRequests(schedule),
-      },
-    })
-  );
+export async function createSchedule(schedule: Schedule) {
+  const dynamoClient = new dynamodb.DynamoDBClient({});
+  try {
+    await dynamoClient.send(
+      new dynamodb.BatchWriteItemCommand({
+        RequestItems: {
+          [configTableName]: scheduleToBatchedPutRequests(schedule),
+        },
+      })
+    );
+  } finally {
+    dynamoClient.destroy();
+  }
```

Comment:
removed need to pass dynamoclient with each schedule request, instead opting to create and destroy clients with each call.

This should help ensure that the clients get cleaned up properly as the number of tests grow (each test will create a new schedule, likely using a different client).